deploy/docker/clickhouse-setup/docker-compose.yaml

```diff
@@ -30,7 +30,7 @@ services:
         condition: service_healthy
     restart: on-failure
     command:
-      - --queryService.url=http://query-service:8080
+      - --queryService.url=http://query-service:9090
```

Comment:
@mindhash This changes has to be added in multiple places.

Do ping me when this PR is merged so that I can include the final changes for the same.